tests/phpunit/TemplateTest.php

```diff
@@ -211,10 +211,16 @@ public function testTemplateRenaming() {
   }
   
   public function testDoiExpansion() {
-    $text = "{{Cite web | http://onlinelibrary.wiley.com/doi/10.1111/j.1475-4983.2012.01203.x/abstract}}";
+    $text = "{{Cite web | http://onlinelibrary.wiley.com/doi/10.1111/j.1475-4983.2012.01203.x/files.pdf}}";
     $prepared = $this->prepare_citation($text);
     $this->assertEquals('cite journal', $prepared->wikiname());
     $this->assertEquals('10.1111/j.1475-4983.2012.01203.x', $prepared->get('doi'));
+    $this->assertNull($prepared->get('url'));
+    $text = "{{Cite web | http://freecopies.com/doi/10.1111/j.1475-4983.2012.01203.x/file.pdf}}";
```

Comment:
Again, a real URL would be useful here, so that we can be sure that we're actually fixing the problem as it exists in the real world (if indeed it does).